jib-maven-plugin/src/test/resources/maven/settings/settings-security.xml

```diff
@@ -0,0 +1,3 @@
+<settingsSecurity>
+  <master>{}X6FMi5KzfGgHeRrZdTiwUdXfVSSqcNdXvVjt3tw7oxs=</master>
```

Comment:
The empty content in the two curly braces seems weird. Shouldn't it be

`{X6FMi5KzfGgHeRrZdTiwUdXfVSSqcNdXvVjt3tw7oxs=}`

?

Reply:
I guess this is correct, given that tests are working.